tests/ssl/client-certs/server/helpers/platform.js

```
@@ -0,0 +1,60 @@
+  const platform = detectPlatform();
+
+  try {
+    switch (platform) {
+      case 'macos':
+        execCommand(`lsof -ti :${port} | xargs kill -9`);
+        break;
+      case 'linux':
+        execCommand(`lsof -ti :${port} | xargs kill -9`);
+        break;
+      case 'windows':
+        const result = execCommandSilent(`netstat -ano | findstr :${port}`);
```

Biome (2.5.3)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

(lint/correctness/noSwitchDeclarations)